### DISCUSSION TITLE
toast hide after redirectRoute

### DISCUSSION THREAD

**Discussion by Delavor** [livewire/flux#145] [Feature Requests]
Unfortunately after redirectRoute toast immediately close, it would be nice if it didn't disappear like in "masmerise/livewire-toaster"

**Comment by patrykszady** [comment]
https://github.com/livewire/flux/issues/91
```
@persist('toast')
    <flux:toast />
@endpersist
```

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

As there is already a discussion #139 open for this, I'm going to close this discussion, so we can continue the conversation there.
